atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceGetRowKeysInRangeTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.keyvalue.cassandra;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertThat;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import com.google.common.collect.ImmutableListMultimap;
+import com.google.common.collect.ImmutableMap;
+import com.palantir.atlasdb.AtlasDbConstants;
+import com.palantir.atlasdb.containers.CassandraResource;
+import com.palantir.atlasdb.keyvalue.api.Cell;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.protos.generated.TableMetadataPersistence;
+import com.palantir.atlasdb.table.description.TableMetadata;
+import com.palantir.common.random.RandomBytes;
+
+import okio.ByteString;
+
+public class CassandraKeyValueServiceGetRowKeysInRangeTest {
+    private static final TableReference GET_ROW_KEYS_TABLE = TableReference.createFromFullyQualifiedName("test.rows");
+    private static final byte[] NO_SWEEP_METADATA = TableMetadata.builder()
+            .sweepStrategy(TableMetadataPersistence.SweepStrategy.NOTHING).build().persistToBytes();
+
+    private CassandraKeyValueService kvs;
+
+    @ClassRule
+    public static final CassandraResource CASSANDRA = new CassandraResource();
+
+    @Before
+    public void initialize() {
+        kvs = CASSANDRA.getDefaultKvs();
+        kvs.createTable(GET_ROW_KEYS_TABLE, NO_SWEEP_METADATA);
+    }
+
+    @After
+    public void cleanup() {
+        kvs.truncateTable(GET_ROW_KEYS_TABLE);
+    }
+
```

Comment:
I think the case missed here is the empty range case from the other class.

Reply:
sure, added this in